src/applications/personalization/profile-2/sass/connected-acct.scss

```diff
@@ -0,0 +1,44 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```

Comment:
nit: this file should probably be called `connected-apps` or something like that.